src/Components/PlayList/PlayList.js

```diff
@@ -0,0 +1,62 @@
+import "./PlayList.css";
+import { useVideoManagement } from "../../Contexts/VideoContextProvider";
+import { Link } from "react-router-dom";
+
+export function PlayList() {
+  const { state } = useVideoManagement();
+  return (
+    <div class="playlist-videos">
+      {state?.currentUser?.playLists?.map((item) => {
+        return (
+          <div class="playlist-video-section">
+            <h2 class="playlist-section-title">{item.name}</h2>
+            {item?.videos?.map((videoItem) => {
+              return (
+                <div class="playlist-video-container">
+                  <Link
+                    to={`/videos/${videoItem._id}`}
+                    class="playlist-thumbnail"
+                  >
+                    <img
+                      class="playlist-thumbnail-image"
+                      src={videoItem.thumbnail}
+                      alt=""
```

Comment:
Give alt attribute a name, try playlist thumbnail
```suggestion
                      alt="playlist thumbnail"
```